src/main/kotlin/net/atos/zac/smartdocuments/rest/RESTMappedSmartDocumentsTemplate.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.smartdocuments.rest
+
+import nl.lifely.zac.util.AllOpen
+import nl.lifely.zac.util.NoArgConstructor
+import java.util.UUID
+
+@NoArgConstructor
+@AllOpen
+data class RESTMappedSmartDocumentsTemplate(
+    var id: String,
```

Comment:
Do we really need `vars` here? Can these not just be`vals`?

Reply:
Yes they can, these are just plain data classes. Adapted tests and usage.

Reply:
I was wrong :( JSON Deserialization fails with NPE. Seems we'll have to stick with `var`

Reply:
ah too bad. I think we ran into this before with other data classes. Surely there must be a way to get it working with `vals` you would think. For a rainy day to have a look at.. :-)